app/src/test/groovy/gov/hhs/cdc/trustedintermediary/ApplicationContextTest.groovy

```diff
@@ -0,0 +1,27 @@
+package gov.hhs.cdc.trustedintermediary.context
```

Comment:
I just realized, this file is in the wrong folder.  It also needs to be added into a `context` sub-folder.

Reply:
Good catch, I imported the class.

Reply:
I meant you should go the other way.  Keep the `.context` on the `package` line and move this file into a newly created `context` sub-folder so it mirrors the `src/main`.